resources/META-INF/plugin.xml

```diff
@@ -12,7 +12,7 @@
   <vendor url="https://github.com/flutter/flutter-intellij">flutter.io</vendor>
 
   <category>Custom Languages</category>
-  
+  <version>33.4.1</version>
```

Comment:
I would remove this line.

Reply:
Done.

Reply:
Just a note, this was added by `plugin generate`.